### PR TITLE
add feature: cycling tabBar

### PR DESCRIPTION
**Pull request by krgoodnews**
Hi, marcostriselli.

I made the Cycling Tab Bar feature.
if set true 'isCyclingEnabled', you can swipe firstVC to lastVC. The opposite is also possible.

Thank you, master.

### CODE REVIEW

**Comment by marcosgriselli** [comment]
@krgoodnews thanks for the PR! This is a really nice feature. Would you mind adding a couple of things? 
- A small section on the readme explaining what `isCyclingEnabled` does
- Add cycling assertions to the UI test  (I can do it or guide you if you're not sure how to tackle it)

**Comment by krgoodnews** [comment]
@marcosgriselli thanks for the comment. I will modify the readme.
I have not done a UI test, but I will try it.

**Comment by krgoodnews** [comment]
Updated Readme.md and added some assertions to the UI Test.
I hope you like it.

Thanks, @marcosgriselli

**Comment by marcosgriselli** [comment]
Perfect, I'll review the PR and test the branch later today

File: README.md [hunk 1]
```diff
@@ -97,6 +97,15 @@ setDiagonalSwipe(enabled: true)
 ```
 --->
 
+### Enable Cycling Swipe Tab Bar
+
+`SwipeableTabBarController` supports cycling switch tab. If you want cycling switch tab, set true `isCyclingEnabled`
```
Comment (marcosgriselli) [inline README.md]:
```suggestion
`SwipeableTabBarController` supports cycling between the first and last tab like a carousel. Simply set `isCyclingEnabled` to `true`.
```

File: README.md [hunk 2]
```diff
@@ -97,6 +97,15 @@ setDiagonalSwipe(enabled: true)
 ```
 --->
 
+### Enable Cycling Swipe Tab Bar
```
Comment (marcosgriselli) [inline README.md]:
```suggestion
### Enable Cycling 
```

**Review by marcosgriselli — CHANGES_REQUESTED** [review]
A few minor changes in documentation. I'm also wondering if there are clearer ways to set up the calculations but I guess I can review that at another time.

**Comment by krgoodnews** [comment]
I think it would be good to have some chance to share our opinions. Thank you 👍

**Review by marcosgriselli — APPROVED** [review]
Perfect, thanks for the changes! Closes #64

**Comment by krgoodnews** [comment]
Thank you. I learned so much from my first contribution.
I wish there will be a better chance with you.

**Pull request merged**